tests/cpu/ee_cop0/performance.expected

```diff
@@ -0,0 +1,13 @@
+-- TEST BEGIN
+setting pcr0: 89abcdef
+setting pcr1: 01234567
+setting pccr: 000ffbfe
+reading from even/odd numbered pcr specifier maps to pcr0/pcr1: 12121212121212121212121212121212
+writing to even/odd numbered pcr specifier maps to pcr0/pcr1: 45454545454545454545454545454545
```

Comment:
Hmm, I wonder if the assembler is just masking the bits when encoding the instruction... but this behavior seems sane.

-[Unnown]